HUBSPOT_TO_S3/php/README.md

```diff
@@ -0,0 +1,36 @@
+## Hubspot to S3 in PHP
+
+A simple example integration that takes a file from S3 and optionally creates companies and/or contacts in HubSpot.
+
+It expects the following environment variables:
+
+```
+    PAN_CFG_S3_BUCKET_NAME
+    PAN_CFG_S3_FILE_NAME
+    PAN_CFG_S3_REGION
+    PAN_CFG_MAKE_CONTACT
+    PAN_CFG_MAKE_COMPANY
+    PAN_SEC_AWS_SECRET_ACCESS_KEY
+    PAN_SEC_AWS_ACCESS_KEY_ID
+    PAN_SEC_HUBSPOT_ACCESS_TOKEN
+```
```

Comment:
I assume these values are listed in the .env.default. Its unnecessary to list them here and makes it annoying to add a new env var because then both the readme and .env.default need to be updated